bot/interfaces.py

```diff
@@ -0,0 +1,95 @@
+"""This script contains the class The911Bot and the interfaces needed to use it.
+>>> class MessageDemo:
+...     def output(self, text):
+...         print text
+>>>
+>>> class ResponseImp (IResponse):
+...     def __init__(self, messageObject):
+...         self.myMessageObject = messageObject
+...     def send(self,message):
+...         self.myMessageObject.output(message)
+>>>
+>>> messageObject = MessageDemo()
+>>> response = ResponseImp(messageObject)
+>>> the911Bot = The911Bot()
+>>> the911Bot.why(response)
+This bot was created by Surge Consulting in response to the tragic events of 2016-08-24. In memory of Simon Hancock.
+>>>
+"""
+
+from slackbot.bot import respond_to
+import store
+
+class IResponse:
+    """Defines an interface for sending a response to the caller.
+    >>> response = IResponse()
```

Comment:
Convention is to not have tests for interfaces, and raise NotImplementedError for functions that need to be overridden. See the explanation here: https://docs.python.org/3/library/exceptions.html#NotImplementedError